less/slider.less

```diff
@@ -17,7 +17,7 @@
         position: absolute;
         width: 50px;
         top: -32px;
-        text-align: center;
+        text-align: center !important;
```

Comment:
Could you duplicate `text-align: center;` down on line 25 so you don't have to use `!important`?